src/Main.ts

```diff
@@ -1347,6 +1384,20 @@ export class Main {
         } // Otherwise, not a known room.
     }
 
+    private async onMatrixPresence(ev: PresenceEvent) {
+        log.debug(`Presence for ${ev.sender}`, ev);
+        const presence = ev.content.presence === "online" ? "auto" : "away";
+        const clients = await this.clientFactory.getClientsForUser(ev.sender);
+        for (const client of clients) {
+            log.debug(`Set presece of user on Slack to ${presence}`);
+            await client.users.setPresence({
+                presence,
+            });
+            // TODO: We need to bridge the status_msg somehow, but nothing in Slack
+            // really fits.
```

Comment:
![Peek 2020-09-18 19-45](https://user-images.githubusercontent.com/10872136/93628851-8b6a9100-f9e7-11ea-8ea7-7a37396c8ef4.gif)

Reply:
Oh I had a long convo with @turt2live about this. I'm not convinced that setting a more permanent status which is not tied to your presence state, is the same as matrix presence status messages which will disappear when your presence changes. Times are tough :s

Reply:
We're deliberately not going to bridge this for the time being, but hopefully Matrix will offer a better way for more permanent statuses.